### PR TITLE
KEYCLOAK-7768 : spring boot2

### PR DESCRIPTION
**Pull request by sebastienblanc**

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]

File: release_notes/topics/4_1_0_final.adoc [hunk 1]
```diff
@@ -1,3 +1,4 @@
-= Test Feature
+= KEYCLOAK-7768 - Making Spring Boot 2 the default starter
+
+From release 4.1 , the Spring Boot Starter will be based on the Spring Boot 2 Adapter. For older Spring Boot version, the starter `keycloak-legacy-spring-boot-starter` is available.
```
Comment (matthewhelmke) [inline release_notes/topics/4_1_0_final.adoc]: Please change to this:
Starting with release 4.1, the Spring Boot starter will be based on the Spring Boot 2 adapter. If you are using an older Spring Boot version, the `keycloak-legacy-spring-boot-starter` is available

**Review by matthewhelmke — DISMISSED** [review]
@mposolda once these changes are made, you have my blessing to approve and merge (I will be out tomorrow).

**Review by mposolda — APPROVED** [review]

**Comment by dalu** [comment]
What does this mean?
I don't see any change in the commit except for documentation.

**Comment by matthewhelmke** [comment]
This is the repo for keycloak documentation. Code changes are in a different repo, usually keycloak/keycloak.

**Comment by dalu** [comment]
Yeah, who would've thought that. But can anyone explain what
<blockquote>
Starting with release 4.1, the Spring Boot starter will be based on the Spring Boot 2 adapter. If you are using an older Spring Boot version, the keycloak-legacy-spring-boot-starter is available.
</blockquote>
If I'm not using Spring Boot at all, does this affect me in any way?
Or does this mean that Keycloak uses Spring Boot 2 instead of 1?

**Comment by sebastienblanc** [comment]
@dalu That does affect you only if you uses the Spring Boot Keycloak adapter / starter in your apps.

**Pull request merged**